xml/System.Windows.Forms/ListBox.xml

```diff
@@ -1970,9 +1970,8 @@
         <Parameter Name="e" Type="System.EventArgs" Index="0" FrameworkAlternate="netcore-3.0;netcore-3.1" />
       </Parameters>
       <Docs>
-        <param name="e">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="e">An <see cref="T:System.EventArgs" /> that contains the event data.</param>
```

Comment:
Since the link to [EventArgs](https://docs.microsoft.com/en-us/dotnet/api/system.eventargs?view=netcore-3.1) will show up next to the param, maybe we could simplify this description into this?:

```suggestion
        <param name="e">An event data instance.</param>
```